schemas/orchestrator.yml

```diff
@@ -83,6 +83,19 @@
       description: >
         Type of resource being acted upon.
 
+    - name: resource.parent_type
```

Comment:
Maybe `resource.parent.type`? It's nice to have flexibility in the future with a `resource.parent.*` namespace.